src/alerts_utils.cc

```diff
@@ -331,6 +329,16 @@ s_alert_load_state_new (zlistx_t *alerts, const char *path, const char *filename
             cursor = zconfig_next (cursor);
             continue;
         }
+
+        const char *encoded_alert_desc = fty_proto_description (alert);
+        size_t encoded_alert_desc_size = strlen (encoded_alert_desc);
+        uint8_t *alert_desc = (uint8_t *) zmalloc (1 + (5 * encoded_alert_desc_size) / 4);
```

Comment:
Is the `1+` for trailing zero in the encoded string, or to be on the safe side of rounding truncation/errors in the division?

Reply:
+1 is for trailing zero  - according to z85 spec